### PR TITLE
Fx mother nrc lookup max date

### PR DESCRIPTION
**Pull request by bennsimon**
#58

### CODE REVIEW

File: opensrp-giz-malawi/src/main/assets/json.form/child_enrollment.json [hunk 1]
```diff
@@ -332,17 +332,18 @@
...
-        "min_date": "01-01-1900",
+        "min_date": "01-01-1960",
         "max_date": "today-10y"
       },
       {
         "key": "nrc_number",
         "openmrs_entity_parent": "",
         "openmrs_entity": "person_attribute",
-        "openmrs_entity_id": "National_ID",
+        "openmrs_entity_id": "nrc_number",
```
Comment (dubdabasoduba) [inline opensrp-giz-malawi/src/main/assets/json.form/child_enrollment.json]: why did you change this?

File: opensrp-giz-malawi/src/main/java/org/smartregister/giz/repository/GizMalawiRepository.java [hunk 2]
```diff
@@ -176,9 +181,24 @@ private void runLegacyUpgrades(SQLiteDatabase database) {
...
     }
 
+    /**
+     * Version 16 added service_group column
+     *
+     * @param database
+     */
+    private void upgradeToVersion16AddServiceGroupColumn(SQLiteDatabase database) {
+        try{
+            database.execSQL(RecurringServiceTypeRepository.ADD_SERVICE_GROUP_COLUMN);
```
Comment (dubdabasoduba) [inline opensrp-giz-malawi/src/main/java/org/smartregister/giz/repository/GizMalawiRepository.java]: Why are you adding this new column?

File: opensrp-giz-malawi/src/main/java/org/smartregister/giz/repository/GizMalawiRepository.java [hunk 3]
```diff
@@ -66,7 +68,7 @@ public void onCreate(SQLiteDatabase database) {
 
         runLegacyUpgrades(database);
 
-        onUpgrade(database, 7, BuildConfig.DATABASE_VERSION);
+        onUpgrade(database, 8, BuildConfig.DATABASE_VERSION);
```
Comment (dubdabasoduba) [inline opensrp-giz-malawi/src/main/java/org/smartregister/giz/repository/GizMalawiRepository.java]: If this migration is necessary upgrade the database version on the giz gradle file

File: opensrp-giz-malawi/src/main/java/org/smartregister/giz/repository/GizMalawiRepository.java (hunk 2 above)
Comment (bennsimon) [reply opensrp-giz-malawi/src/main/java/org/smartregister/giz/repository/GizMalawiRepository.java]: Its required by immunization module

File: opensrp-giz-malawi/src/main/assets/json.form/child_enrollment.json (hunk 1 above)
Comment (bennsimon) [reply opensrp-giz-malawi/src/main/assets/json.form/child_enrollment.json]: it should be okay....
